src/utils/saml.ts

```diff
@@ -486,17 +753,75 @@ export const preValidateResponse: PreValidateResponseT = (
       .item(0);
 
     if (Response) {
```

Comment:
you can early return here ie. `if (!Response) { }`

Reply:
fixed